R/tt_compatibility.R

```diff
@@ -374,6 +391,19 @@ rbindl_rtables <- function(x, gap = 0, check_headers = TRUE) {
 
     lapply(x, function(xi) chk_compat_cinfos(x[[1]], xi)) ##col_info(xi)))
 
+    rbind_annot <- list(title = "", subtitles = character(), footer = character(), pf = character())
```

Comment:
Please use the slot names here as names of the list (ie prov_footer instead of pf) for ease of understanding.

Also, we need to talk about whether these checks should be independent or collective. My instinct is collective, but lets discuss it at our next standup. @anajens @edelarua @ayogasekaram @Melkiades please think on this and come with your thoughts collected.

Reply:
I think as @edelarua thought it respects the original user request and makes sense to keep only the first annotation or all equal independently anyway. Doing this collectively adds things to explain we do not need to. In other words, as you said in other context, this rbind checks would need to be smart and we do not want this too much. Even if we develop an intuitive strategy to do this collectively (which would add checks and complexity), it will drop information more than it does now